app/models/cart_item.rb

```diff
@@ -0,0 +1,6 @@
+class CartItem < ApplicationRecord
+    belongs_to :cart, optional: true
```

Comment:
I think here, a cart item is only created when i add a lesson to cart. Don't think it can be optional.